clang/test/dpct/memcpy_optimization.cu

```diff
@@ -0,0 +1,17 @@
+// RUN: dpct --format-range=none --optimize-migration -out-root %T/memcpy_optimization %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck --match-full-lines --input-file %T/memcpy_optimization/memcpy_optimization.dp.cpp %s
+#include <cuda_runtime.h>
+
+int main(){
+
+    float *a, *b;
+//CHECK:  /*
+//CHECK:  DPCT1114:{{[0-9]+}}: There is no synchronization operation append after memcpy. This is based on the assumption that the source host memory is pageable. If this assumption doesn't hold, please append .wait() after memcpy for correct synchronization behavior.
```

Comment:
cudaMemcpy is migrated to asynchronization memcpy, assuming in original code the source host memory is pageable memory. If this assumption doesn't hold, please append ‘.wait()’ after memcpy to  synchronize.